src/sbt-test/sbt-header/simple/build.sbt

```diff
@@ -0,0 +1,21 @@
+import de.heikoseeberger.sbtheader.license.Apache2_0
```

Comment:
Please use the names I am using in sbt-fresh:
  - call the build file `test.sbt`
  - call the task according to what it is doing
  - also use the task marco
  - don't use outdated operators (I guess you have learned from Eugene's old blog post)

Here is the example:
```
import org.eclipse.jgit.api.Git
import org.eclipse.jgit.storage.file.FileRepositoryBuilder
import scala.collection.JavaConverters._

val checkInitialCommit = taskKey[Unit]("Verify the initial commit")
checkInitialCommit := {
  val git = new Git((new FileRepositoryBuilder).setWorkTree(baseDirectory.value).build())
  val commits = git.log().call().asScala.toVector
  assert(commits.size == 1, "There must be exactly one commit!")
}
```